src/mainwindow.cpp

```diff
@@ -4096,6 +4096,21 @@ void MainWindow::on_actionExportEDL_triggered()
 void MainWindow::on_actionExportFrame_triggered()
 {
     if (!MLT.producer() || !MLT.producer()->is_valid()) return;
+    if (Settings.proxyEnabled()) {
```

Comment:
It is not ideal to check the setting because not all clips in a project have a proxy when proxy mode is on. Instead, check `producer.get_int(kIsProxyProperty)`. However, you might have to try to get the producer from the current shared frame; `MLT.producer()` is not the correct thing to check.

Reply:
Good idea. I will try that.

Reply:
That was pretty slick. Now it only prompts if the frame is actually from a proxy file. Thanks for the suggestion.